src/sass/_functions.scss

```diff
@@ -229,3 +229,40 @@ Will generate for example following classes:
         @include createSelector(#{$newSelector}, $properties);
     }
 }
+
+// @include breakpoint('xs') { width: 90%; }
+@mixin breakpoint($bpName, $breakpoints: $breakpoints) {
+    $bpValue: map-get($breakpoints, $bpName);
+    @media (min-width: $bpValue) { @content; }
+}
+
+@mixin createClasses($map) {
+    @each $selector, $definition in $map {
+        .#{$selector} {
```

Comment:
Creating class has been done already in a mixin (with a poor name :) ). That can be called and avoid code duplication.

```
@mixin createSelector()
```